core/domain/opportunity_services.py

```diff
@@ -146,25 +145,19 @@ def _create_exploration_opportunity_summary(topic, story, exploration):
         ExplorationOpportunitySummary. The exploration opportunity summary
         object.
     """
-
-    audio_language_codes = set(
-        language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES)
-
-    complete_translation_languages = set(
+    complete_translation_language_list = (
         exploration.get_languages_with_complete_translation())
-
+    language_codes_needing_voice_artists = set(
```

Comment:
@sagangwee Think it's worth adding a TODO+issue here?

Reply:
Done.

Reply:
Er wait -- to be clear, I meant for this line, i.e. updating the logic for language codes needing voice artists.

Reply:
Ah, sorry. Updated.